app/app.go

```diff
@@ -373,7 +392,45 @@ func NewOsmosisApp(
 	app.mm.RegisterInvariants(app.CrisisKeeper)
 
 	app.configurator = module.NewConfigurator(app.AppCodec(), app.MsgServiceRouter(), app.GRPCQueryRouter())
-	app.mm.RegisterServices(app.configurator)
+	err = app.mm.RegisterServices(app.configurator)
+	if err != nil {
+		panic(err)
+	}
+
+	// UNFORKING v2 TODO: Verify that the NewBasicManagerFromManager call is correct.
+	// Notice I have to override the gov ModuleBasic with all the custom proposal handers, otherwise we lose them in the CLI.
+	app.ModuleBasics = module.NewBasicManagerFromManager(
+		app.mm,
+		map[string]module.AppModuleBasic{
+			"gov": gov.NewAppModuleBasic(
+				[]govclient.ProposalHandler{
+					paramsclient.ProposalHandler,
+					// UNFORKING v2 TODO: Verify it is okay to remove these
+					// upgradeclient.LegacyProposalHandler,
+					// upgradeclient.LegacyCancelProposalHandler,
+					poolincentivesclient.UpdatePoolIncentivesHandler,
+					poolincentivesclient.ReplacePoolIncentivesHandler,
+					// UNFORKING v2 TODO: Verify it is okay to remove these
```

Comment:
.

Reply:
This should be fine: https://github.com/cosmos/ibc-go/blob/v8.0.0/docs/docs/05-migrations/11-v7-to-v8.md#governance-v1-migration